src/SFA.DAS.EmployerCommitmentsV2.Web/Models/CreateCohort/ConfirmProviderViewModel.cs

```diff
@@ -1,10 +1,15 @@
 ﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 
 namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort
 {
     public class ConfirmProviderViewModel : IndexViewModel
     {
         public long ProviderId { get; set; }
+        public string ProviderName { get; set; }
+
+        [Required(ErrorMessage = "Select a training provider")]
```

Comment:
Is this needed if we are using FluentValidation?

Reply:
Removed annotation